contrib/yggdrasil/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "yggdrasil"
+pkgver = "0.5.5"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./cmd/yggdrasil/", "./cmd/yggdrasilctl/"]
+make_check_args = list(make_build_args)
+hostmakedepends = ["go"]
+pkgdesc = "Experiment in scalable routing as an encrypted IPv6 overlay network"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "LGPL-3.0-only"
+url = "https://yggdrasil-network.github.io"
+source = f"https://github.com/yggdrasil-network/yggdrasil-go/archive/v{pkgver}.tar.gz"
+sha256 = "cdbb56b19b91b828afa282554862efb2a79dd4ada26dfb5a7bf3b0c5220f6c17"
+
+
+def pre_build(self):
+    self.do("rm", "build")
+
+
+def do_build(self):
+    self.golang.build(env={"PKGNAME": pkgname, "PKGVER": pkgver})
```

Comment:
`make_build_env =` instead of overriding do_build should work for this

Reply:
actually no, this doesn't work now either. these variables are read by `./build` to write go ldflags. nothing else reads the env, and we invoke go directly

Reply:
Good catch, thanks. I'll set the ldflags instead then